mandatory/2-function-creation.js

```diff
@@ -5,8 +5,16 @@ Complete the function to check if the variable `num` satisfies the following req
 - is less than or equal to 100
 Tip: use logical operators
 */
-
-function validate(num) {}
+function validate(num) {
+  return typeof num === "number" && num % 2 === 0 && num <= 100;
```

Comment:
honestly Dawit I cannot understand this solution , we want to check if (num) is number and even and less or equal to 100, but u just return these as they are already facts without checking it ,while you have the right solution below this commented out !